python/paddle/fluid/dygraph/dygraph_to_static/convert_call_func.py

```diff
@@ -95,13 +110,14 @@ def dyfunc(x):
           #  [1. 1. 1.]]
 
     """
+    translator_logger.log(1, "Convert call: convert {}.".format(func))
     func_self = None
```

Comment:
`call` is an AST but not clear concept for users.  Adding `ast.Call` or `Convert callable object:`

Reply:
Thanks. use `Convert callable object`:
`translator_logger.log(1, "Convert callable object: convert {}.".format(func))`